### PR TITLE
feat(info): show new DC available info message

### PR DESCRIPTION
**Pull request by radireddy**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

## Title of the Pull Requests 
Show info message about the availability of new data centers in the APAC region

### Description of the Change
Added info message to show the availability of new data centers in the APAC region. This message will appear in PCI home page until the user dismisses the message or it is removed manually after one month.

### Applicable Issues
MBE-247

### CODE REVIEW

File: client/app/home/home.controller.js [hunk 1]
```diff
@@ -25,6 +28,43 @@ class HomeCtrl {
         return this.guides.sections;
       });
   }
+
+  onNewRegionsAvailableMessageDismiss() {
+    this.dismissNewRegionsAvailableKey(this.newRegionsAvailableMessageKey);
+  }
+
+  initNewRegionsAvailableMessage(key) {
+    this.getNewRegionsAvailableKey(key)
```
Comment (FredericEspiau) [inline client/app/home/home.controller.js]: `getNewRegionsAvailableKey` returns a `Promise`, so you should return its return value

File: client/app/home/home.html [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+            data-ng-if="$ctrl.showNewRegionsAvailableMessage">
+            {{ ::'homepage_apac_regions_availability_message'| translate }}
+        </oui-message>
+        <div class="text-center">
+            <h1 class="d-inline">
+                <span data-translate="homepage_welcome_to_your_manager_1"></span>
+                <span data-translate="homepage_welcome_to_your_manager_2"></span>
+            </h1>
+            <p class="lead"
+            data-ng-bind-html="'homepage_top_guides_header' | translate: { url: $ctrl.guides.all }">
```
Comment (FredericEspiau) [inline client/app/home/home.html]: Indentation

File: client/app/home/home.html [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
-           data-ng-bind-html="'homepage_top_guides_header' | translate: { url: $ctrl.guides.all }">
-        </p>
+    <div class="mt-5 py-5">
+        <oui-message data-type="info"
+            data-on-dismissed="$ctrl.onNewRegionsAvailableMessageDismiss()"
+            data-ng-if="$ctrl.showNewRegionsAvailableMessage">
+            {{ ::'homepage_apac_regions_availability_message'| translate }}
+        </oui-message>
+        <div class="text-center">
+            <h1 class="d-inline">
```
Comment (FredericEspiau) [inline client/app/home/home.html]:
```suggestion
            <h1>
```

File: client/app/home/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -5,5 +5,6 @@
   "homepage_type_of_guide_vps": "VPS",
   "homepage_type_of_guide_pci": "Public Cloud",
   "homepage_scheduled_maintenance_operation": "Une opération de maintenance est programmée pour la région BHS1 le 19 septembre 2018 entre 3:00 et 8:00 UTC-4. Aucune perturbation n'est à prévoir sur vos instances, seules certaines APIs de cette région seront injoignables pendant la durée de l'opération",
-  "homepage_scheduled_maintenance_operation_more_detail": "Plus de détail sur la tâche travaux correspondante:"
-}
\ No newline at end of file
+  "homepage_scheduled_maintenance_operation_more_detail": "Plus de détail sur la tâche travaux correspondante:",
+  "homepage_apac_regions_availability_message": "Nouvelles regions Public Cloud disponibles: Singapour (SGP) et Sydney, Australie (SYD)"
```
Comment (FredericEspiau) [inline client/app/home/translations/Messages_fr_FR.json]:
```suggestion
  "homepage_apac_regions_availability_message": "Nouvelles régions Public Cloud disponibles : Singapour (SGP) et Sydney (SYD)"
```

File: client/app/home/home.controller.js (hunk 1 above)
Comment (radireddy) [reply client/app/home/home.controller.js]: not required. if the key is not found, show the message on UI and create it in the background, no need to wait for its status.

Comment (FredericEspiau) [reply client/app/home/home.controller.js]: `return`ing `Promise`s sends a message that this function can be chained, it doesn't have to have a real meaning

File: client/app/home/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -5,5 +5,6 @@
   "homepage_type_of_guide_vps": "VPS",
   "homepage_type_of_guide_pci": "Public Cloud",
   "homepage_scheduled_maintenance_operation": "Une opération de maintenance est programmée pour la région BHS1 le 19 septembre 2018 entre 3:00 et 8:00 UTC-4. Aucune perturbation n'est à prévoir sur vos instances, seules certaines APIs de cette région seront injoignables pendant la durée de l'opération",
-  "homepage_scheduled_maintenance_operation_more_detail": "Plus de détail sur la tâche travaux correspondante:"
-}
\ No newline at end of file
+  "homepage_scheduled_maintenance_operation_more_detail": "Plus de détail sur la tâche travaux correspondante:",
```
Comment (antleblanc) [inline client/app/home/translations/Messages_fr_FR.json]:
```suggestion
  "homepage_scheduled_maintenance_operation_more_detail": "Plus de détail sur la tâche travaux correspondante :",
```

Comment (radireddy) [reply client/app/home/translations/Messages_fr_FR.json]: Hi got a specific request to add this space, can you please refer MBE-247. Even I prefer not to have space.

Comment (antleblanc) [reply client/app/home/translations/Messages_fr_FR.json]: @radireddy Yes you can safely add this space. It's a french grammar rule;)

**Comment by jleveugle** [comment]
Hello @radireddy,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request merged**